src/test/resources/constructorcall-type/TestHttpStub.java

```diff
@@ -0,0 +1,16 @@
+package com.test;
+
+public class TestHttpStub {
+
+  private static com.ibm.ws.webservices.engine.description.OperationDesc _getInformationOperation0 =
+      null;
+
+  private static void _getInformationOperation0() {
+    com.ibm.ws.webservices.engine.description.FaultDesc[] _faults0 =
+        new com.ibm.ws.webservices.engine.description[] {};
+    _getInformationOperation0 =
+        new com.ibm.ws.webservices.engine.description.OperationDesc("getVerificationInformation",
+            new javax.xml.namespace.QName("", "getVerificationInformation"), _faults0,
+            "http://com.gov//getVerificationInformation");//constructor call which needs to be transformed
+  }
+}
```

Comment:
This can be greatly simplified, all we need is two types that are not on the classpath.

```suggestion
class ConstructorCallWithTypesNotOnClasspath {
    public static void main(String[] args) {
        type.not.on.Classpath instanceOfTypeNotOnClasspath = null;
        new another.type.not.on.Classpath(instanceOfTypeNotOnClasspath);
    }
}
```

Reply:
Changed it. This greatly increases the chance someone else understands the test case in the future. One day, some will have enough time to write a tool to reduce java ASTs with a failing test case like C reduce(https://embed.cs.utah.edu/creduce/)

Reply:
That'd be pretty neat.